core/lib/multivm/src/versions/vm_latest/bootloader/state.rs

```diff
@@ -114,6 +120,17 @@ impl BootloaderState {
         self.subversion
     }
 
+    pub(crate) fn get_preexisting_blocks_number(&self) -> usize {
+        max(self.l2_blocks.len(), 1) - 1
+    }
```

Comment:
nit: (subjectively) `self.l2_blocks.len().saturating_sub(1)` is easier to read.
Can you please also elaborate on how it's used and why it needs to be like this i.e. return 0 for len=0 and len-1 in other cases?